src/FunctionApps/python/IntakePipeline/function.json

```diff
@@ -2,14 +2,11 @@
   "scriptFile": "__init__.py",
   "bindings": [
     {
-      "authLevel": "Anonymous",
-      "type": "httpTrigger",
+      "name": "blob",
+      "type": "blobTrigger",
       "direction": "in",
-      "name": "req",
-      "methods": [
-        "get",
-        "post"
-      ]
+      "path": "bronze/decrypted/ELR/{name}",
```

Comment:
Is this path correct? Don't we want he blob trigger to trigger whenever there is a new blob for any message type? What about point the trigger to just bronze/decrypted?

Reply:
This was an accidental remnant of running tests and pushing before changing the path back. The code has been tested on all three file types and an updated `path` has been pushed to the PR.